scripts/mkcloud

```diff
@@ -354,7 +360,7 @@ EOF
     env|grep -e ^debug_ -e ^pre_ -e ^vlan_ -e ^want_ -e ^net_ -e ^nodenumber -e ^clusterconfig -e ^ironicnetmask | sort >> $mkcconf
 
     cp -a $mkcconf $artifacts_dir/
-    $scp -r $SCRIPTS_DIR $mkcconf root@$adminip:
+    $scp -r $SCRIPTS_DIR $mkcconf root@${ip_wrap_left}$adminip${ip_wrap_right}:
```

Comment:
Perhaps we just want to enclose the admin IP in [] and call it done rather than this madness?

Reply:
yeah so something things need the wrapping, others don't. So we can't just change $adminip straight. $ip_wrap_left/right aren't that great either. How about we either:

1. Create an $adminip_wrapped for cases adminip is used in URIs, or
2. Create a wrap_ip(ip) function. though this might not look nice inline "scp root@$(wrap_ip $adminip):...

Maybe I'll go with 1 for now, and make the creation happen when the left and right do.

Reply:
I like the idea of a format_ip function that returns [$ip] if it's IPv6 and $ip if it's IPv4. $adminip_wrapped=$adminip for IPv4 works for me too.